client/state/themes/themes-ui/test/actions.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { hideThemesBanner } from '../actions';
+import { THEMES_BANNER_HIDE } from 'state/action-types';
+
+describe( 'actions', () => {
+	describe( 'hideThemesBanner()', () => {
+		test( 'Should return the expected action object', () => {
+			const action = hideThemesBanner();
+			expect( action ).to.eql( {
```

Comment:
We try to avoid `chai` in favor of Jest's built-in test helpers now.  Try dropping the `chai` import, an changing this line to `expect( action ).toEqual( {` (see https://facebook.github.io/jest/docs/en/expect.html#toequalvalue).